full-stack-tests/core/src/frontend/map/PlanarClipMask.test.ts

```diff
@@ -206,7 +207,8 @@ describe("Planar clip mask (#integration)", () => {
     });
   });
 
-  it("is masked by priority by dynamic geometry", async () => {
+  it("is masked by priority by dynamic geometry", async function () {
+    this.timeout(480000);
```

Comment:
Put a comment explaining why, so we can come back and revert when/if we fix whatever caused the tests to start timing out more frequently.